src/test/java/com/auth0/client/auth/AuthAPITest.java

```diff
@@ -944,6 +939,171 @@ public void shouldCreateLogInWithClientCredentialsGrantRequest() throws Exceptio
         assertThat(response.getExpiresIn(), is(notNullValue()));
     }
 
+    // Login with Passwordless
+
+    @Test
+    public void shouldCreaateStartEmailPasswordlessFlowRequest() throws Exception {
+        Request<PasswordlessEmailResponse> request = api.startPasswordlessEmailFlow("user@domain.com",
+                PasswordlessEmailType.CODE);
+        assertThat(request, is(notNullValue()));
+
+        server.jsonResponse(PASSWORDLESS_EMAIL_RESPONSE, 200);
+        PasswordlessEmailResponse response = request.execute();
+        RecordedRequest recordedRequest = server.takeRequest();
+
+        assertThat(recordedRequest, hasMethodAndPath("POST", "/passwordless/start"));
+        assertThat(recordedRequest, hasHeader("Content-Type", "application/json"));
+
+        Map<String, Object> body = bodyFromRequest(recordedRequest);
+        assertThat(body, hasEntry("connection", (Object) "email"));
+        assertThat(body, hasEntry("client_id", (Object) CLIENT_ID));
+        assertThat(body, hasEntry("client_secret", (Object) CLIENT_SECRET));
+        assertThat(body, hasEntry("email", (Object) "user@domain.com"));
+
+        assertThat(response, is(notNullValue()));
+        assertThat(response.getEmail(), not(isEmptyOrNullString()));
+        assertThat(response.getId(), not(isEmptyOrNullString()));
+        assertThat(response.isEmailVerified(), is(notNullValue()));
+    }
+
+    @Test
+    public void startPasswordEmailFlowShouldThrowWhenEmailIsNull() throws Exception {
```

Comment:
should these be "passwordless" instead of "password"? repeats on the others
```suggestion
    public void startPasswordlessEmailFlowShouldThrowWhenEmailIsNull() throws Exception {
```